cypress/integration/test_ui.spec.js

```diff
@@ -0,0 +1,9 @@
+describe('Test UI elements', () => {
```

Comment:
This test does not seem to test the flow and selects according to text on page. Can we remove this file?